services/static-webserver/client/source/class/osparc/Preferences.js

```diff
@@ -98,6 +98,22 @@ qx.Class.define("osparc.Preferences", {
       init: "always",
       event: "changeWalletIndicatorVisibility",
       apply: "__patchPreference"
+    },
+
+    userInactivityThreshold: {
+      check: "Number",
+      nullable: false,
+      init: 1800,
+      event: "changeUserInactivityThreshold",
+      apply: "__patchPreference"
+    },
+
+    jobConcurrencyLimit: {
```

Comment:
Is there any reason I can't change this value? Does something need changing to enable it?